test/e2e/network/network_policy.go

```diff
@@ -1515,8 +1524,14 @@ var _ = SIGDescribe("NetworkPolicy [LinuxOnly]", func() {
 				framework.ExpectNoError(err, "Error occurred while getting pod status.")
 			}
 
-			podServerAllowCIDR := fmt.Sprintf("%s/24", podServerStatus.Status.PodIP)
-			podServerIP := fmt.Sprintf("%s/32", podServerStatus.Status.PodIP)
+			allowMask := 24
+			exceptMask := 32
+			if utilnet.IsIPv6String(podServerStatus.Status.PodIP) {
+				allowMask = 64
+				exceptMask = 128
+			}
+			podServerAllowCIDR := fmt.Sprintf("%s/%d", podServerStatus.Status.PodIP, allowMask)
+			podServerIP := fmt.Sprintf("%s/%d", podServerStatus.Status.PodIP, exceptMask)
 			// Exclude podServer's IP with an Except clause
 			podServerExceptList := []string{podServerIP}
```

Comment:
It's not really an "IP" when it has a prefix length. Maybe just get rid of the `podServerIP` variable and do this like in the above test

Reply:
yep